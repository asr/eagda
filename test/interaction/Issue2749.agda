{-# OPTIONS --no-unicode #-}

module asciionly where

id : {A : Set} -> A -> A
id = {!!}
