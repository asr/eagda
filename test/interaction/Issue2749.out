(agda2-info-action "*Error*" "Issue2749.agda:3,8-17 The name of the top level module does not match the file name. The module asciionly should be defined in one of the following files: asciionly.agda asciionly.lagda ../asciionly.agda ../asciionly.lagda ../asciionly.agda ../asciionly.lagda asciionly.agda asciionly.lagda agda-default-include-path/asciionly.agda agda-default-include-path/asciionly.lagda" nil)
((last . 3) . (agda2-maybe-goto '("Issue2749.agda" . 38)))
(agda2-highlight-load-and-delete-action)
(agda2-status-action "")
(agda2-info-action "*Error*" "Error: First load the file." nil)
(agda2-highlight-load-and-delete-action)
(agda2-status-action "")
